www/success.html

```diff
@@ -10,19 +10,31 @@
       type="text/css"
       href="https://unpkg.com/@patternfly/patternfly@4.96.2/patternfly.min.css"
     />
+    <script lang="javascript">
+      function onLoad() {
+        window.location.href = 'podman-desktop:///preferences/authentication-providers'
```

Comment:
I think this URL is not handled/managed by Podman Desktop and I'm not sure it'll be this URL being used.

Either we use a shorter URL that is accepted/handled property
or we need to add inside Podman Desktop a way to handle /preferences/authentication-providers but in that case I think it might be missing a prefix

Reply:
I see now, basically I can have podman-desktop:/// and it would give the same result.

Reply:
> we need to add inside Podman Desktop a way to handle /preferences/authentication-providers but in that case I think it might be missing a prefix

Do you mean a prefix similar to `podman-desktop://extension/` used in extension installation?  

Would something like `podman-desktop://goto/preferences/authentication-providers` work? So we can translate it to router.goto('/preferences/authentication-providers')` call on podman desktop side.